code/game/objects/buckling.dm

```diff
@@ -69,6 +66,10 @@
 		if(buckle_prevents_pull)
 			M.pulledby.stop_pulling()
 
+	if(length(M.grabbed_by))
+		for(var/obj/item/grab/G in M.grabbed_by)
+			qdel(G)
```

Comment:
Remove it from the list first, then `qdel` it, so there's not null entries in the list---unless grabs already clean this up themselves (I forget, you'll have to check).

Reply:
https://github.com/ParadiseSS13/Paradise/blob/657e78b7e0b9ba1aecdd6d0e6f02a1702326c229/code/modules/mob/mob_grab.dm#L433-L440

Seems to be the case

Reply:
Yeah, I kinda figured.

Last thing to do then is to just make this:

```
		for(var/obj/item/grab/G in M.grabbed_by)
			qdel(G)
```

Unless this is a list that can be null---but I have doubts.

Reply:
```
/mob
	[...]
	var/list/grabbed_by = list()
```

As per [mob_defines.dm, L97](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/mob_defines.dm#L97), this list is defined at the `/mob` level on start-up and a quick search for `grabbed_by =` doesn't return anything else, so it's safe to assume that list can't be null